app/experimenter/settings.py

```diff
@@ -409,7 +409,7 @@
 KINTO_COLLECTION_NIMBUS_DESKTOP = "nimbus-desktop-experiments"
 KINTO_COLLECTION_NIMBUS_MOBILE = "nimbus-mobile-experiments"
 KINTO_COLLECTION_NIMBUS_PREVIEW = "nimbus-preview"
-
+KINTO_ADMIN_URL = config("KINTO_ADMIN_URL", default=urljoin(KINTO_HOST, "/v1/admin/"))
```

Comment:
Host already has v1 in it https://github.com/mozilla/experimenter/blob/main/.env.sample#L27